src/app/modules/users/add-single-user/add-single-user.component.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  <mat-toolbar role="toolbar" class="task-header">
+    <mat-icon style="color: white;">account_circle</mat-icon>
+    <h5 class="title">Add New User</h5>
+  </mat-toolbar>
+</div>
+<mat-dialog-content class="mat-typography custom-dialog-content">
+  <div class="row">
+    <dynamic-form [fields]="formdata" (submit)="submit($event)">
+    </dynamic-form>
+  </div>
+</mat-dialog-content>
+<mat-dialog-actions class="custom-dialog-actions" align="end">
+  <button mat-button mat-dialog-close>CANCEL</button>
+  <button mat-button style="background: antiquewhite;" (click)="onSubmit()" cdkFocusInitial color="primary">ADD NEW USER</button>
```

Comment:
Primary should work

Reply:
if i change primary some other designs are getting effected